src/compiler/checker.ts

```diff
@@ -16743,6 +16746,65 @@ namespace ts {
             }
         }
 
+        function markDecoratorMedataDataTypeNodeAsReferenced(node: TypeNode): void {

```

Comment:
Just for consistency `markDecoratorMedataDataTypeNodeAsReferenced` should be `markDecoratorMedataDataTypeNodeAsReference`